### DISCUSSION TITLE
Shiny for DSA

### DISCUSSION THREAD

**Discussion by vikachooo** [HTAhackathon2024/.github#23] [Ideas]
### What is the problem we're trying to solve

I remember struggling with creating an interactive dashboard with DSA tornado diagram in Excel (last summer, before I discovered R).

My idea is to create a shiny app where we can adjust values for parameters and see how it affects ICER. We can do both a one-way tornado diagram and a graph for multi-way DSA.

### How will we tackle the problem?

1. Select R packages 
2. Generate a Markov model
3. Conceptualise Shiny
4. Consider exploring other DSA methods from this [paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7790801/ )

### Which tools will we use?

Shiny, heemod, any other packages with DSA functions?

**Comment by n8thangreen** [comment]
Looks like this is the code for this paper
https://github.com/Vreman/DSA/blob/master/Script

Perhaps shiny-fy this code?

**Reply by vikachooo** [reply]
Shiny-fy, exactly 😀
